model/model-verifier/src/test/java/io/spine/model/verify/ModelVerifierTest.java

```diff
@@ -120,6 +155,28 @@ void failOnDuplicateHandlers() {
         assertThrows(DuplicateCommandHandlerError.class, () -> verifier.verify(spineModel));
     }
 
+    @Disabled
+    //TODO:1/4/2019:Serhii Lekariev:disabled until https://github.com/SpineEventEngine/core-java/issues/932 is resolved
```

Comment:
This should be `@Dicabled("...")` and the TODO. Also, please align your TODO template along with the other team members' templates.